__tests__/app.test.js

```diff
@@ -262,3 +269,66 @@ describe("Post /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/comments/:comment_id", () => {
+  const newVote = { inc_votes: 20 };
+  test("Status 200 - updates the given articles' votes and returns it", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          body: "I find this existence challenging",
+          votes: 120,
+          topic: "mitch",
+          author: "butter_bridge",
+          created_at: expect.any(String),
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("Status 400 - incomplete request body ", () => {
+    const newVote = {};
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid post body");
+      });
```

Comment:
if inc_votes is missing, the article should not be updated and we could expect the original single object to be returned. A status code of 200 would be more of a robust test for this.